DatadogSessionReplay/Sources/Processor/SRDataModelsBuilder/Wireframes.swift

```diff
@@ -0,0 +1,404 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+import UIKit
+
+/// The border properties of this wireframe. The default value is null (no-border).
+@_spi(Internal) public struct ShapeBorder {
```

Comment:
I'm wondering if it would make more sense to expose it as a protocol that SR objects are implementing 🤔

Slightly less boilerplate.

Reply:
Could you elaborate a bit as to what this would look like and what we'd gain from it? 
It's a bit unclear for me.

Reply:
I guess inits "for free"